### ISSUE TITLE
Custom colors on Plone.de

### ISSUE THREAD

**Issue by Molochem** [plone/plone.de#42]
@tisto @fredvd @jonaspiterek @ericof @sneridagh @jackahl 

The Staging Instance is now using VLT, and all that is needed is the Migration of all Grid Blocks that were previously a custom Variant.

It is Time to move on to selecting custom colors for things like Teaser-Flags, Buttons etc. as mentioned in #35.

Plone-Blue (#0086C3) seems to be a given, other Suggestions are welcome

**Comment by fredvd** [comment]
I think the plone-blue is the only really important color here.  The slider on the plone.org homepage with the darker blue(?) color is maybe a nice alternative to have in the background. And when you wan to select larger background blocks 

And there is the 'normal' row alternating background color also on plone.org which is already implemented: #f9f9f9

<img width="932" alt="banner plone_org logo 6" src="https://github.com/plone/plone.de/assets/394784/b83f8b23-2a57-49e2-92af-ea6d3214ce9b">

**Comment by jonaspiterek** [comment]
For now we will ship plone.de without custom colors, VLT should not be customized heavily. I will leave the issue open for a later discussion.

**Comment by tisto** [comment]
@jonaspiterek @fredvd @Molochem I agree. Let's ship the new VLT to live asap. Then we can fix issues we might face. Then we can think about customizations, like the right Plone blue color.

I would like to use plone.de as an example how to use VLT without any heavy customizations. The background color or color option should be a JSON configuration, not a customization with shadow components. cc @sneridagh @danalvrz

**Comment by sneridagh** [comment]
@tisto We worked this out in the upcoming version of VLT, we moved the color definitions to its own registry setting. It can be backported (we did in .com) not a big deal, but of course, we would like this to be OOTB.
